### PR TITLE
Update cdk from v1 to v2

### PR DESCRIPTION
**Pull request by JMHReif**
*Issue #, if available:* n/a. With the existing example code, was getting a message at the bottom of the deployment to upgrade the project to use CDK v2. Wanted to migrate the code and get rid of that warning.

*Description of changes:* Updating the example code to use AWS CDK v2 (instead of the v1).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: infrastructure/pom.xml [hunk 1]
```diff
@@ -35,26 +35,22 @@
         <!-- AWS Cloud Development Kit -->
         <dependency>
             <groupId>software.amazon.awscdk</groupId>
-            <artifactId>core</artifactId>
-            <version>${cdk.version}</version>
+            <artifactId>aws-cdk-lib</artifactId>
```
Comment (pankajagrawal16) [inline infrastructure/pom.xml:38]: Can you please use the latest version of cdk lib here

**Review by pankajagrawal16 — COMMENTED** [review]
Thanks for the contribution, left two minor suggestions, once its resolved I can merge

**Comment by JMHReif** [comment]
@pankajagrawal16, thank you for reviewing! I've made the changes. Let me know if there's anything else.

**Review by pankajagrawal16 — APPROVED** [review]
Looks good now. Guess you will have to resolve conflicts first.

Thanks again for contributing👍

**Comment by JMHReif** [comment]
Fixed! :)

**Pull request merged**